Server/server.js

```diff
@@ -335,16 +336,19 @@ app.delete('/deleteEvent/:userId/:eventId', async (request, response) => {
 
 // -------------------------------------------------------------------------------------------------
 //User Events Table 
-app.get('/userEventsTable', async (request, response) => {
+app.get('/userEventsTable/:userId', async (request, response) => {
```

Comment:
This could be something to think about when it comes to testing your server, but you could create a routes folder with different files of routes (ex: one dedicated for event specific requests, another for apis, etc). So when it's time for testing, your testing files and suites can be more specific to be within the same user flow. 

This could also improve organization among your routes, making it easier to find a specific route if you needed to update it.